### ISSUE TITLE
Generator loss problem.

### ISSUE THREAD

**Issue by Tinghh** [ChengBinJin/V-GAN-tensorflow#17]
![Uploading image.png…]()
gan loss and seg loss stand for what loss?I know that GAN network has generator loss and discriminator loss. Why is g-loss in the code so expressed as:
 self.g_loss = self.alpha_recip * gan_loss + seg_loss.

**Comment by ChengBinJin** [comment]
> gan loss and seg loss stand for what loss?I know that GAN network has generator loss and discriminator loss. Why is g-loss in the code so expressed as:
> self.g_loss = self.alpha_recip * gan_loss + seg_loss.

The seg loss means a segmentation loss. The generator has two tasks, one is to cheat the discriminator, the second is to classify each pixel of the input as the supervised segmentation task.
